src/simulation/navigation/simpleNav/simpleNav.h

```diff
@@ -49,6 +49,14 @@ class SimpleNav: public SysModel {
     void writeOutputMessages(uint64_t Clock);
 
 public:
+    double gyroStandardDeviation=1E-5;    //!< Standard deviation for each rate component
```

Comment:
:large_orange_diamond: Any value on making these setters/getters? I know the rest of the module doesn't have them but I can't immediately see a reason to not head in that direction. 🤔

Reply:
should I clean it up and make setters/getters for everything?

Reply:
We'll do this in a separate ticket